app/routes/sign-in-oidc.js

```diff
@@ -25,11 +25,14 @@ module.exports = {
     }
   },
   handler: async (request, h) => {
-    const response = await getAccessToken(request.payload.code)
-    const state = decodeState(request.payload.state)
+    validateState(request, request.query.state)
+    const state = decodeState(request.query.state)
+    const response = await getAccessToken(request.query.code)
+    validateInitialisationVector(request, response.access_token)
     const redirect = getRedirectPath(state.redirect)
+    clearCache(request)
+    cacheRefreshToken(request, response.redirect_token)
```

Comment:
Updating so refresh token is stored in cache to keep cookie sizes low and to support refresh logic.